src/katgpucbf/send.py

```diff
@@ -44,9 +45,9 @@ class DescriptorSender:
     first_interval
         Delay (in seconds) immediately after starting. If not specified, it
         defaults to `interval`.
-    all_substreams
-        Send descriptors to all substreams if true, otherwise default behaviour
-        is to send only to the first substream.
+    substreams
+        Substream indexes to which descriptors are sent. If not specified,
```

Comment:
```suggestion
        Substream indices to which descriptors are sent. If not specified,
```
Just for bike-shedding purposes, ignore at your preference.